commit ed5e92950851eef60385ec2f624dee65fb855716
Author: Angela Fong <angela.fong@joyent.com>
Date:   2019-08-15T17:36:02-07:00 (8 weeks ago)
    
    TRITON-1872 Incorrect location echoed in releng/ls-missing-release-builds script

diff --git a/tools/releng/ls-missing-release-builds b/tools/releng/ls-missing-release-builds
index 5b2f7f5..6321109 100755
--- a/tools/releng/ls-missing-release-builds
+++ b/tools/releng/ls-missing-release-builds
@@ -113,6 +113,6 @@ echo "$targets" | while read target buildisprivate; do
     latest=$(mls $latest_mpath 2>/dev/null || true)
     if [[ -z "$latest" ]]; then
         echo "# $target, '$latest_mpath' does not exist" >&2
-        echo "./tools/launch-build ${PLAT_FLAVOR_ARG} -b $release $target"
+        echo "./tools/releng/launch-build ${PLAT_FLAVOR_ARG} -b $release $target"
     fi
 done
